app/src/ui/thank-you/thank-you.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react'
+
+import { encodePathAsUrl } from '../../lib/path'
+
+import { ReleaseNote } from '../../models/release-notes'
+
+import { Dialog, DialogContent } from '../dialog'
+
+import { RichText } from '../lib/rich-text'
+import { Repository } from '../../models/repository'
+import { getDotComAPIEndpoint } from '../../lib/api'
+import { GitHubRepository } from '../../models/github-repository'
+import { Owner } from '../../models/owner'
+
+// HACK: This is needed because the `Rich`Text` component
+// needs to know what repo to link issues against.
+// Since release notes are Desktop specific, we can't
+// rely on the repo info we keep in state, so we've
+// stubbed out this repo
+const desktopOwner = new Owner('desktop', getDotComAPIEndpoint(), -1)
+const desktopUrl = 'https://github.com/desktop/desktop'
+const desktopRepository = new Repository(
+  '',
+  -1,
+  new GitHubRepository('desktop', desktopOwner, -1, false, desktopUrl),
+  true
+)
+
+const ReleaseNoteHeaderLeftUri = encodePathAsUrl(
+  __dirname,
+  'static/release-note-header-left.svg'
+)
+const ReleaseNoteHeaderRightUri = encodePathAsUrl(
+  __dirname,
+  'static/release-note-header-right.svg'
+)
+
+interface IThankYouProps {
+  readonly onDismissed: () => void
+  readonly emoji: Map<string, string>
+  readonly userContributions: ReadonlyArray<ReleaseNote>
+  readonly friendlyName: string
+  readonly latestVersion: string | null
+}
+
+export class ThankYou extends React.Component<IThankYouProps, {}> {
+  private renderList(
+    releaseEntries: ReadonlyArray<ReleaseNote>
+  ): JSX.Element | null {
+    if (releaseEntries.length === 0) {
+      return null
+    }
+
+    const options = new Array<JSX.Element>()
+
+    for (const [i, entry] of releaseEntries.entries()) {
+      options.push(
+        <li key={i}>
+          <RichText
+            text={entry.message}
+            emoji={this.props.emoji}
+            renderUrlsAsLinks={true}
+            repository={desktopRepository}
+          />
+        </li>
+      )
+    }
+
+    return (
+      <div className="section">
+        <ul className="entries">{options}</ul>
+      </div>
+    )
+  }
+
+  private renderConfetti(): JSX.Element | null {
```

Comment:
I am not 100% sure. But, we would likely see better performance if we did this outside the renderer. Grab a div ref and pump the confetti in. Currently, if user moves their mouse during confetti, there can sometimes be a noticeable lag.